ocs_ci/ocs/fiojob.py

```diff
@@ -33,24 +33,21 @@
 logger = logging.getLogger(__name__)
 
 
-def get_storageutilization_size(target_percentage, ceph_pool_name):
+def get_ceph_storage_stats(ceph_pool_name):
```

Comment:
this function has the potential to be used not only as part of fiojob. How about placing it somewhere else, like cluster.py?

Reply:
One can import this function from anywhere, but I don't think this is very likely. Neither I think that it's a good idea to move it.